inc/modules/themeoptions/class-pdfoptions.php

```diff
@@ -1663,7 +1634,7 @@ static function scssOverrides( $scss ) {
 		if ( isset( $options['pdf_image_resolution'] ) ) {
 			$resolution = $options['pdf_image_resolution'];
```

Comment:
We should probably remove this whole block. If someone has saved `pdf_image_resolution` and this stays, they will be stuck because the option can't be changed via the interface anymore.